processor/src/main/java/com/tickaroo/tikxml/processor/generator/CodeGeneratorHelper.kt

```diff
@@ -92,19 +97,78 @@ class CodeGeneratorHelper(val customTypeConverterManager: CustomTypeConverterMan
                 Long::class.java.canonicalName to "java.lang.Long"
         )
 
-        fun generatePrimitiveConverter(
+        fun tryGeneratePrimitiveConverter(
```

Comment:
Why does this methodname starts with "try"?

Reply:
It can return null, if the map doesn't contain type name.
In read/write part (where this is called) it will generate different assignment when this return value is null.

Reply:
👍 I forgot about that.

Yeah that part of the code is actually not nice, but it wasn't nice before either, so yeah ...